extensions/binding/org.eclipse.smarthome.binding.ntp/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ntp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ntp">
+		<label>Date</label>
+		<description>NTP Updated Date &amp; Time</description>

```

Comment:
I couldn't call it a NTP server, as certainly I'm not serving the network
Time Protocol, it is a NTP client.
If you consider that confusing/incorrect, instead I could call it a Time
and Date provider or something

2015-10-04 22:02 GMT+02:00 Kai Kreuzer <notifications@github.com>:

> In
> extensions/binding/org.eclipse.smarthome.binding.ntp/ESH-INF/thing/thing-types.xml
> <https://github.com/eclipse/smarthome/pull/448#discussion_r41101678>:
>
> > @@ -0,0 +1,55 @@
> > +<?xml version="1.0" encoding="UTF-8"?>
> > +<thing:thing-descriptions bindingId="ntp"
> > +	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
> > +	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
> > +	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
> > +
> > +	<thing-type id="ntp">
> > +		<label>Date</label>
> > +		<description>NTP Updated Date &amp; Time</description>
>
> why client and not server? The Thing that provides the info is the server
> itself, you only implemented a client within your binding to get the time
> from it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eclipse/smarthome/pull/448/files#r41101678>.
>